### DISCUSSION TITLE
ImageGlass Kobe 8.8.3.28

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1515] [General]
<img width="640" alt="ImageGlass 8.8.3.28" src="https://raw.githubusercontent.com/ImageGlass/website-content/main/releases/screenshots/v8.8/8.8_1.jpg">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/8.8.3.28/total?color=%23016485&label=downloads&style=for-the-badge)](https://imageglass.org/download)

## 👐🏿 Support ImageGlass Kobe on Microsoft Store!
This is the paid version, you will need to pay a small amount after 7-day trial. You will get the convenience of fast, easy installation onto all of your Windows devices along with fully automatic, behind-the-scenes updates with all the newest features, improvements, and fixes.

Alternatively, you can use the one-time donation from [♥ GitHub Sponsor](https://github.com/sponsors/d2phap), then drop me a message in [ImageGlass Discord](https://discord.com/channels/818852544859209748/818852544859209751) or [email](https://imageglass.org/about), and I will send the code shortly.

<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH?cid=GitHubRelease&referrer=appbadge&source=GithubD2phap">
  <img height="69" src="https://user-images.githubusercontent.com/3154213/197670614-b9ecc804-ebb2-44cc-88c3-5dd321564a51.svg" />
</a>

## ⚠ ImageGlass End User License Agreement (EULA) update!
Starting from **April 1st, 2023 and ImageGlass version 8.8**, it is now required for users to register for a free license in order to use ImageGlass for non-personal purposes. This update has been announced on the [ImageGlass website](https://imageglass.org/news/imageglass-end-user-license-agreement-eula-update-81), and users can find more information about the new End User License Agreement (EULA) update at https://imageglass.org/license.

### 🌱 New:
- Adds setting `DisabledMenus` in `igconfig.xml` to disable ImageGlass features using menu name (#1498).
- Adds support for relative path of Exiftool in `igconfig.xml` file (#1208).
- Adds support for displaying and extracting Unicode characters from EXIF metadata (#1169).
- Adds a new confirmation dialog when the user overrides the current image to warn them about the limitations of ImageGlass regarding image saving (#1094).
- The About dialog now includes information about the ImageGlass [license](https://imageglass.org/license) and [privacy policy](https://imageglass.org/privacy).
- The Check for Update feature has been improved to show more information about the update and check if the user's system can install version 9.0 in the future.
- Image saving has been updated to only work with supported formats listed in the "Write" column of [the supported formats](https://imageglass.org/docs/supported-formats) page (#1504):
  + If the user attempts to save an unsupported format, the "Save as..." dialog will be opened instead.
  + This change also affects other image saving-related features such as rotation, flipping, and cropping. 

### 🔮 Improvements:
- Upgraded Magick.NET to version 13.0.0.
- Improved loading speed for animated `WEBP` format (#1466) by directly using `libwebp` v1.3.0.
- Thumbnails are now automatically centered (#1049).
- Enhanced the corner radius of the toolbar, thumbnail bar, and menu to ensure optimal display on high DPI screens running Windows 11.
- Implemented rounded frameless window borders on Windows 11.

### 🐛 Bugfixes:
- Fixed an issue where the user couldn't minimize the window in Window Fit mode (#1448).
- Fixed a bug that prevented ImageGlass from loading certain TIF files with invalid tags (#1181, #1454).
- Fixed an issue where the Exif tool was not functional when using the default name of ExifTool: `exiftool(-k).exe` (#1178).
- Fixed an issue where images became corrupted during the save process, causing display errors when navigating to the next image or exiting the app (#1097, #1273, #1504).
- Fixed an issue where the "loading..." message always shows when viewing next images.

### 🥒 Checksums:

File | SHA-1
-- | --
ImageGlass_Kobe_8.8.3.28_x64.msi | `137de9afcb8a26792b83f6296acd10cff90013f7`
ImageGlass_Kobe_8.8.3.28_x64.zip | `3d808e3f8348155b692713b3471fe919bce6097c`
ImageGlass_Kobe_8.8.3.28_x86.msi | `fb7e006cbc247441e94c9ff85492f1302d95c4ec`
ImageGlass_Kobe_8.8.3.28_x86.zip | `7a932ecf83951f114d0edf6f925ce8286183cf84`

## 🛣 ImageGlass 9.0 development updates
- [Version 9.0 beta 1](https://imageglass.org/news/announcing-imageglass-moon-9-0-beta-1-75) was already released.
- [Version 9.0 beta 2](https://imageglass.org/news/announcing-imageglass-moon-9-0-beta-2-78) was already released.
- [Version 9.0 beta 3](https://imageglass.org/news/announcing-imageglass-moon-9-0-beta-3-80) was already released. Try it and let me know your feedback.
- [Version 9.0 beta 4](https://github.com/d2phap/ImageGlass/projects/14) is under development.
- From version 9.0, .NET 6 Runtime is required to run ImageGlass, you can download it at https://dotnet.microsoft.com/en-us/download/dotnet/6.0.

**Full Changelog**: https://github.com/d2phap/ImageGlass/compare/8.7.11.6...8.8.3.28

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/8.8.3.28'>ImageGlass Kobe 8.8.3.28</a>.</em>

**Comment by d2phap** [comment]
## ♨ Updates
- Updated the binary files (and hashcode) to fix the issue #1516: Zooming by hotkey in Window Fit mode does not work

**Reply by d2phap** [reply]
## ♨ Updates
- Updated the license to make the registration for non-personal purposes optional, see https://imageglass.org/license

**Reply by BoomerangAide** [reply]
Don't you mean "Updated the license to make the registration for personal purposes optional," or "Updated the license to make the registration for non-commercial purposes optional," or something?

**Reply by d2phap** [reply]
to make the registration optional, see https://github.com/d2phap/ImageGlass/discussions/1505#discussioncomment-5479841
